doc/rtd/explanation/format.rst

```diff
@@ -121,6 +193,48 @@ The :command:`make-mime` subcommand takes pairs of (filename, "text/" mime
 subtype) separated by a colon (e.g., ``config.yaml:cloud-config``) and emits a
 MIME multipart message to :file:`stdout`.
 
+User data format to Mime Content type mapping
+---------------------------------------------
+
+We use the following format to outline the content-type the user is
+using
+
+.. code-block::
+
+   Content-Type: text/<content-type>
+
+Below is a mapping of a specific user data format to the `content-type`
+value to use when using the mime multipart archive format.
+
+
++----------------------------+----------------------------+
+| User data                  | Content Type               |
++============================+============================+
+| cloud boothook             | cloud-boothook             |
++----------------------------+----------------------------+
+| cloud config               | cloud-config               |
++----------------------------+----------------------------+
+| cloud config archive       | cloud-config-archive       |
++----------------------------+----------------------------+
+| cloud config jsonp         | cloud-config-jsonp         |
++----------------------------+----------------------------+
+| Jinja2                     | jinja2                     |
++----------------------------+----------------------------+
+| User data script           | x-shellscript              |
++----------------------------+----------------------------+
+| include file               | x-include-url              |
++----------------------------+----------------------------+
+| include file once          | x-include-once-url         |
++----------------------------+----------------------------+
+| Part handler               | part-handler               |
++----------------------------+----------------------------+
+| x-shellscript-per-instance | x-shellscript-per-instance |
++----------------------------+----------------------------+
+| x-shellscript-per-once     | x-shellscript-per-once     |
++----------------------------+----------------------------+
+| x-shellscript-per-boot     | x-shellscript-per-boot     |
++----------------------------+----------------------------+
+
```

Comment:
I think i might just be dense or have overlooked something, but I am unsure of what to do with this information now. 

Where do I put `Content-Type: text/<content-type>`? I just don't know how or where this is used. "content-type" doesn't seem to appear really anywhere else in these docs.